review/core/admin.py

```diff
@@ -28,11 +33,16 @@ def get_reviewee(self, obj):
 
 
 class PersonReviewAdmin(admin.ModelAdmin):
-    list_display = ('reviewer', 'reviewee')
+    list_display = ('round', 'reviewer', 'reviewee')
 
 
 class ManagerProjectCommentAdmin(admin.ModelAdmin):
-    list_display = ('get_project', 'manager', 'get_reviewee')
+    list_display = ('get_round', 'get_project', 'manager', 'get_reviewee')
+
+    def get_round(self, obj):
+        return obj.project_review.round
+
+    get_round.short_description = 'Round'
```

Comment:
can we use `verbose_name` in models instead of this way?

https://docs.djangoproject.com/en/3.1/ref/models/fields/#verbose-name

Reply:
I believe `verbose_name` refers to the name of the field when you're defining it **in the model**. But in code above, we are defining a function in the **`ModelAdmin`**.

Reply:
Yes. but if we define verbose name in the **`Model`**, there is no need to define these names in **`ModelAdmin`**